bin/improver-create-grid-with-halo

```diff
@@ -49,13 +49,13 @@ def main():
                         "containing data on a source grid.")
     parser.add_argument('output_file', metavar='OUTPUT_FILE', help="NetCDF "
                         "file defining the target grid with additional halo.")
-    parser.add_argument('--halo_radius', metavar='HALO_RADIUS', default=162.,
-                        type=float, help="Size of halo (in km) with which to "
+    parser.add_argument('--halo_radius', metavar='HALO_RADIUS', default=162000,
+                        type=float, help="Size of halo (in m) with which to "
                         "pad the input grid.  Default is 162 km.")
```

Comment:
I think 162km default should be 162000m to match the intended units.

Reply:
I did actually leave this on purpose - I think 162 km is much more readable than 162000m.  But can change if it's a major issue.

Reply:
I only worry that someone reading quickly might use the wrong units. I'll leave it to Stephen to make a final decision.